client/tmclient/screens.py

```diff
@@ -41,7 +41,7 @@ def show_menu(self):
                 menu_button('login', lambda _:self.show_login()),
                 menu_button('create a new user account', lambda _:self.show_register()),
                 sub_menu(self, 'settings', [
-                    menu_button('forget login details', lambda _:True),
+                    menu_button('forget login details', lambda _:self.forget_details()),
```

Comment:
is this change meant to be in here, too?

Reply:
huh, that must have gotten stuck in there from some other branch, oops.